nym-vpn-core/crates/nym-vpnd/src/logging.rs

```diff
@@ -6,17 +6,23 @@ use tracing_appender::non_blocking::WorkerGuard;
 use crate::service;
 
 pub fn setup_logging() {
-    let filter = tracing_subscriber::EnvFilter::builder()
-        .with_default_directive(tracing_subscriber::filter::LevelFilter::INFO.into())
-        .from_env()
-        .unwrap()
-        .add_directive("hyper::proto=info".parse().unwrap())
-        .add_directive("netlink_proto=info".parse().unwrap());
-
-    tracing_subscriber::fmt()
-        .with_env_filter(filter)
-        .compact()
-        .init();
+    #[cfg(any(target_os = "ios", target_os = "macos"))]
```

Comment:
this will not be compiled on iOS I assume?

Reply:
Ah good question, I'm not 100% sure, but if for iOS we won't be using this daemon implementation, then it's probably not necessary to specify ios target macro here.

Reply:
I think so too.